pkg/kubectl/cmd/drain.go

```diff
@@ -495,7 +495,9 @@ func (o *DrainOptions) evictPods(pods []api.Pod, policyGroupVersion string, getP
 				if err == nil {
 					break
 				} else if apierrors.IsTooManyRequests(err) {
-					time.Sleep(5 * time.Second)
+					retryInterval := 5
```

Comment:
I was thinking does the `IsTooManyRequests` can also include other API calls except `eviction`?